### DISCUSSION TITLE
Can not load .obj file's color

### DISCUSSION THREAD

**Discussion by JiayunjieJYJ** [bulletphysics/bullet3#4121] [General]
This is the obj file:
![image](https://user-images.githubusercontent.com/44229958/145932284-2bb734bd-7f4e-4089-a0f6-f5052b5f8fb2.png)

But when I load it in pybullet:  
![image](https://user-images.githubusercontent.com/44229958/145932407-7a21034a-534a-4068-bd74-595b92e9810f.png)

even I use flags = URDF_USE_MATERIAL_COLORS_FROM_MTL:
`p.loadURDF("Wood_Table.urdf", (1, 1, 1), (1,1,1,1),flags = p.URDF_USE_MATERIAL_COLORS_FROM_MTL)`
![image](https://user-images.githubusercontent.com/44229958/145932525-18e4065b-6e63-408b-b021-04363e950760.png)

It just can't load the right color.  

Any suggestions? Thanks!!!

**Comment by JiayunjieJYJ** [comment]
Maybe it is a problem of obj files.

**Comment by rohit-kumar-j** [comment]
@JiayunjieJYJ can you send the obj,mtl files and the script?
